Regression/WarpX-tests.ini

```
@@ -3952,16 +3952,16 @@ useOMP = 1
 numthreads = 1
 analysisRoutine = Examples/Tests/particle_thermal_boundary/analysis_2d.py
 
-#[openbc_poisson_solver]
-#buildDir = .
-#inputFile = Examples/Tests/openbc_poisson_solver/inputs_3d
-#runtime_params = warpx.abort_on_warning_threshold = high
-#dim = 3
-#addToCompileString = USE_OPENPMD=TRUE USE_FFT=TRUE
-#cmakeSetupOpts = -DWarpX_DIMS=3 -DWarpX_FFT=ON -DWarpX_OPENPMD=ON
-#restartTest = 0
-#useMPI = 1
-#numprocs = 2
-#useOMP = 1
-#numthreads = 1
-#analysisRoutine = Examples/Tests/openbc_poisson_solver/analysis.py
+[openbc_poisson_solver]
+buildDir = .
+inputFile = Examples/Tests/openbc_poisson_solver/inputs_3d
+runtime_params = warpx.abort_on_warning_threshold = high
+dim = 3
+addToCompileString = USE_OPENPMD=TRUE USE_FFT=TRUE
+cmakeSetupOpts = -DWarpX_DIMS=3 -DWarpX_FFT=ON -DWarpX_OPENPMD=ON -DCMAKE_BUILD_TYPE=Debug
```

Remove DEBUG build for now.
```suggestion
cmakeSetupOpts = -DWarpX_DIMS=3 -DWarpX_FFT=ON -DWarpX_OPENPMD=ON
```